docs/mkdocs/docs/api/macros/json_diagnostics.md

```diff
@@ -26,11 +26,14 @@ When the macro is not defined, the library will define it to its default value.
 
 ## Notes
 
-!!! danger "ABI incompatibility"
+!!! note "ABI compatibility"
 
-    As this macro changes the definition of the `basic_json` object, it MUST be defined in the same way globally, even
-    across different compilation units: `basic_json` objects with differently defined `JSON_DIAGNOSTICS` macros are
-    not compatible!
+    As of version 3.11.0 this macro is no longer required to be defined consistently throughout a codebase
+    to avoid One Definition Rule (ODR) violations, as the value of this macro is encoded in the namespace
+    of the types, resulting in distinct types. This is intended to allow different libraries to use different
```

Comment:
```suggestion
    to avoid One Definition Rule (ODR) violations, as the value of this macro is encoded in the namespace, resulting in distinct symbol names. This is intended to allow different libraries to use different
```